core/Session.php

```diff
@@ -190,7 +190,14 @@ public static function getSameSiteCookieValue()
     {
         $config = Config::getInstance();
         $general = $config->General;
-        if (!empty($general['enable_framed_pages']) && ProxyHttp::isHttps()) {
+
+        $module = Common::getRequestVar('module', false);
```

Comment:
just fyi you can use `Piwik::getModule()` and `Piwik::getAction()`. not too important though

Reply:
modified